src/components/Follow.tsx

```diff
@@ -0,0 +1,26 @@
+import { Button, Link } from "@mui/material";
+import { h } from "preact";
+import { Follow } from "../types";
+
+export default function Follow({
+  summary,
+  actor,
+  object
+}: Follow) {
+  const acceptFollow = () => {
+    fetch(`/authors/${object}/followers/${actor}`, { method: 'PUT' })
```

Comment:
when I make a call to this through the frontend, it is unable to read the objects as strings returning a `403` error